californium-core/pom.xml

```diff
@@ -37,6 +37,35 @@
 	<build>
 		<plugins>
 			<plugin>
+				<groupId>org.apache.felix</groupId>
+				<artifactId>maven-bundle-plugin</artifactId>
+				<extensions>true</extensions>
+				<configuration>
+					<instructions>
+						<Export-Package>
+							org.eclipse.californium.core,
```

Comment:
I also thought about replacing netty-all with the required individual artifacts. But I was not sure about the changes caused for the current consumers, as it reduces the dependencies (IMHO something good) but also means consumers would need to change their pom.xml files in case they need the whole netty-all and until now relied on getting it transitively.